frontend/src/views/Transfers/components/TransferView.jsx

```diff
@@ -0,0 +1,109 @@
+import PropTypes from 'prop-types'
+import {
+  TransferDetailsCard,
+  CommentList,
+  AttachmentList,
+  AddPlainComment
+} from '.'
+import BCBox from '@/components/BCBox'
+import { Typography, List, ListItem } from '@mui/material'
+import { faCircle } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { decimalFormatter } from '@/utils/formatters'
+import { demoData } from './demo'
+import { useTranslation } from 'react-i18next'
+
+export const TransferView = ({
+  fromOrgId,
+  fromOrganization,
+  toOrgId,
+  toOrganization,
+  quantity,
+  pricePerUnit,
+  transferStatus,
+  isGovernmentUser,
+  totalValue,
+  handleCommentChange,
+  comment
+}) => {
+  const { t } = useTranslation(['common', 'transfer'])
+  return (
+    <>
+      <TransferDetailsCard
+        fromOrgId={fromOrgId}
+        fromOrganization={fromOrganization}
+        toOrgId={toOrgId}
+        toOrganization={toOrganization}
+        quantity={quantity}
+        pricePerUnit={pricePerUnit}
+        transferStatus={transferStatus}
+        isGovernmentUser={isGovernmentUser}
+      />
+      {/* Transfer Details View only */}
+      <BCBox
+        variant="outlined"
+        p={2}
+        mt={2}
+        sx={{
+          backgroundColor: 'transparent.main'
+        }}
+      >
+        <Typography variant="body4">
+          <b>{fromOrganization}</b>
+          {t('transfer:transfers')}
+          <b>{quantity}</b>
+          {t('transfer:complianceUnitsTo')} <b>{toOrganization}</b>
+          {t('transfer:for')}
+          <b>${decimalFormatter({ value: pricePerUnit })}</b>
+          {t('transfer:complianceUnitsPerTvo')}
+          <b>${decimalFormatter(totalValue)}</b> CAD.
+        </Typography>
+      </BCBox>
+      -- demo data --
```

Comment:
Let's remove these demo data markers. just hide the CommentList, AttachmentList, and transactionHistory. We have prs coming for two of those 3.

Reply:
removed and commented.